modules/ROOT/pages/plugins/premium/a11ychecker.adoc

```diff
@@ -269,15 +255,15 @@ NOTE: The table `summary` attribute was deprecated in HTML 5, both the *what* an
 
 * *Notification level (severity)*: Error
 * *https://www.w3.org/TR/WCAG21/#conformance[WCAG level]*: Level A ; Level AA ; Level AAA
-* *<<a11ychecker_html_version,HTML version>>*: HTML4 and HTML5
+* *xref:a11ychecker_html_version[HTML version]*: HTML4 and HTML5
 * *WCAG 2.1 specification*: https://www.w3.org/WAI/WCAG21/Techniques/html/H63.html[H63 - Using the scope attribute to associate header cells and data cells in data tables].
 
 == Configuration Options
 
 The following configuration options affect the behavior of the {pluginname} plugin.
 
 :includedSection: a11yPlugin
-#include::partial$configuration/a11y_advanced_options.adoc[]
+include::partial$configuration/a11y_advanced_options.adoc[]
```

Comment:
```suggestion
include::partial$configuration/a11y_advanced_options.adoc[leveloffset=+1]
```
I missed this when reviewing the other day sorry, but the nav/heading levels isn't correct so we need to adjust this via the include:
![image](https://user-images.githubusercontent.com/1575550/160022376-12db7a46-5bdc-4d78-bbbf-f5ca5ce6919c.png)